content/webapp/components/IIIFItem/IIIFItem.tsx

```diff
@@ -106,14 +161,14 @@ const IIIFItem: FunctionComponent<ItemProps> = ({
       );
     case item.type === 'Video' && !exclude.includes('Video'):
       return (
-        <>
+        <div className="video">
```

Comment:
Is this class added for styling purposes?

Reply:
yeah, it's part of the temporary styling